pom.xml

```diff
@@ -19,7 +19,7 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>3.2.5.Final</quarkus.platform.version>
-        <quarkus.ide.version>3.2.5.Final</quarkus.ide.version>
+        <quarkus.ide.version>3.2.6.Final</quarkus.ide.version>
         <quarkus.qe.framework.version>1.3.0.Beta26</quarkus.qe.framework.version>
         <quarkus-qpid-jms.version>2.4.0</quarkus-qpid-jms.version>
         <apache-httpclient-fluent.version>4.5.14</apache-httpclient-fluent.version>
```

Comment:
Is there a reason why to change only quarkus IDE and not quarkus BOM? I would rather see this two changes together.

Reply:
Yeah, I actually meant to change platform version, thanks

Reply:
Done.